lib/galaxy/tool_util/lint.py

```diff
@@ -89,59 +124,67 @@ def __init__(self, level, skip_types=None, object_name=None):
         self.skip_types = skip_types or []
         self.level = level
         self.object_name = object_name
-        self.found_errors = False
-        self.found_warns = False
+        self.message_list = []
+
+    @property
+    def found_errors(self):
```

Comment:
All these properties are easy to annotate like:
```suggestion
    def found_errors(self) -> bool:
```
Not a big deal but it would be nice :)